ha/core/event_analyzer/filter/filter.py

```diff
@@ -174,7 +174,6 @@ def __init__(self):
         Init method
         """
         super(ClusterResourceFilter, self).__init__()
-        ConfigManager.init("event_analyzer")
```

Comment:
Why is this removed ?

Reply:
This is not required here now. It will be called from EventAnalyzer class where init will happen